.github/workflows/integ_test_api_rest_user_pool.yml

```diff
@@ -36,6 +53,7 @@ jobs:
           xcode_path: '/Applications/Xcode_14.3.app'
 
   api-rest-user-pool-test-tvOS:
+    if: ${{ github.event_name != 'workflow_dispatch' || inputs.ios }}
```

Comment:
should be `tvOS` instead of `ios`

Reply:
Addressed in [457931f](https://github.com/aws-amplify/amplify-swift/pull/3318/commits/457931f2c406914c6dc6409d3de17cc2c818225d)